plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterRating.php

```diff
@@ -145,16 +147,16 @@ function ( $rating ) use ( $selected_rating, $attributes ) {
 		);
 
 		$wrapper_attributes = array(
-			'data-wc-interactive'  => wp_json_encode( array( 'namespace' => $this->get_full_block_name() ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
-			'data-wc-context'      => wp_json_encode(
+			'data-wp-interactive'  => wp_json_encode( array( 'namespace' => $this->get_full_block_name() ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
```

Comment:
Don't forget to simplify the namespace, it's just such an easy and nice clean up:

`data-wp-interactive' => $this->get_full_block_name()`